homeassistant/components/uptimerobot/sensor.py

```diff
@@ -46,7 +46,7 @@ async def async_setup_entry(
             coordinator,
             SensorEntityDescription(
                 key=str(monitor.id),
-                name=monitor.friendly_name,
+                name=None,
```

Comment:
This is a fixed None for all entities now, and thus can be set on the `UptimeRobotSensor` as a class variable instead.